src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -1413,7 +1454,7 @@ private static IEnumerable<object[]> SerializableObjects()
             }
 
             // Extension of core serializable types
-            // Disabling on full framework until we compile against net471
+            // Disabling on full framework until we compile against >=net471
```

Comment:
@ViktorHofer maybe we can enable this now, we're now compiling against net472

Reply:
yes please :)